fendermint/vm/interpreter/src/chain.rs

```diff
@@ -250,12 +255,23 @@ where
                     )
                     .await
                     .context("failed to commit finality")?;
+                    tracing::debug!(
+                        "chain interpreter committed topdown finality, prev_height: {}, prev_finality: {})",
+                        prev_height,
+                        prev_finality.as_ref().map(|f| format!("{f}")).unwrap_or_else(|| String::from("None"))
+                    );
```

Comment:
I do recommend structure logging instead of string interpolation.

Reply:
I changes as much as possible